src/implementations/twig/components/link/link.story.js

```diff
@@ -26,86 +25,110 @@ const withParagraph = (story) => {
   return typeof demo === 'string' ? storyAsString(demo) : storyAsNode(demo);
 };
 
-const iconsList = {};
-iconsList.none = '';
-
-uiIcons.forEach((icon) => {
-  iconsList[icon] = icon;
-});
+const iconsList = uiIcons.reduce((a, b) => {
+  a[b] = b;
+  return a;
+}, {});
+
+const getArgTypes = (data) => {
+  return {
```

Comment:
I like this return statement, I think we should align existing migrated component